.github/workflows/build.yml

```diff
@@ -140,18 +146,22 @@ jobs:
         run: |
           cmake -S . -B ${{ env.BUILD_DIR }} -DCMAKE_INSTALL_PREFIX=${{ env.INSTALL_DIR }} -DCMAKE_BUILD_TYPE=${{ inputs.build_type }} -DLauncher_PORTABLE=OFF -G Ninja
 
-
       - name: Configure CMake on Windows portable 
         if: runner.os == 'Windows' && matrix.portable == true
         shell: msys2 {0}
         run: |
           cmake -S . -B ${{ env.BUILD_DIR }} -DCMAKE_INSTALL_PREFIX=${{ env.INSTALL_DIR }} -DCMAKE_BUILD_TYPE=${{ inputs.build_type }} -G Ninja 
 
       - name: Configure CMake on Linux
-        if: runner.os == 'Linux'
+        if: runner.os == 'Linux' && matrix.portable != true
         run: |
           cmake -S . -B ${{ env.BUILD_DIR }} -DCMAKE_INSTALL_PREFIX=/usr -DCMAKE_BUILD_TYPE=${{ inputs.build_type }} -DLauncher_PORTABLE=OFF -G Ninja
 
+      - name: Configure CMake on Linux Portable 
+        if: runner.os == 'Linux' && matrix.portable == true
+        run: |
+          cmake -S . -B ${{ env.BUILD_DIR }} -DCMAKE_INSTALL_PREFIX=/usr -DCMAKE_BUILD_TYPE=${{ inputs.build_type }} -G Ninja
```

Comment:
This should be `-DCMAKE_INSTALL_PREFIX=${{ env.INSTALL_DIR }}` not `/usr`

Reply:
oh yeah fixed